scripts/run-smoke-test.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# Requires docker and the following ENV vars:
+#
+# PORT_CLIENT_ID
+# PORT_CLIENT_SECRET
+# PORT_BASE_URL (optional, defaults to 'https://api.getport.io')
+#
+
+SCRIPT_BASE="$(cd -P "$(dirname "$0")" && pwd)"
+ROOT_DIR="$(cd -P "${SCRIPT_BASE}/../" && pwd)"
+
+RANDOM_ID="${RANDOM}${RANDOM}" # DOGE - RANDOM
+INTEGRATION_IDENTIFIER="smoke-test-integration-${RANDOM_ID}"
+BLUEPRINT_DEPARTMENT="fake-department-${RANDOM_ID}"
+BLUEPRINT_PERSON="fake-person-${RANDOM_ID}"
+BLUEPRINT_IDENTIFIERS="${BLUEPRINT_DEPARTMENT} ${BLUEPRINT_PERSON}"
+FAKE_INTEGRATION_VERSION=${FAKE_INTEGRATION_VERSION:-0.1.1-dev}
+PORT_BASE_URL_FOR_DOCKER=${PORT_BASE_URL}
+
+if [[ ${PORT_BASE_URL} =~ localhost ]]; then
+    PORT_BASE_URL_FOR_DOCKER=${PORT_BASE_URL//localhost/host.docker.internal}
+fi
+
+TEMP_DIR=$(mktemp -d -t smoke-test-integration.XXXXXXX)
```

Comment:
what is this for?

Reply:
This is where I save the altered `blueprints.json` and `app-config-port.yml`

Reply:
it's also mounted to the container